aten/src/ATen/templates/RegisterSchema.cpp

```diff
@@ -4,10 +4,9 @@
 
 namespace at {
 TORCH_LIBRARY(aten, m) {
-  ${schema_registrations};
-
   // Distributed Ops
   // Implementations located in torch/csrc/jit/runtime/register_distributed_ops.cpp
   m.def("get_gradients(int context_id) -> Dict(Tensor, Tensor)");
 }
+${schema_registrations}
```

Comment:
?

Reply:
Need to bring this out since the namespace is a variable now and I don't want to lose the registration of "get_gradients".